### DISCUSSION TITLE
manual sys prompt

### DISCUSSION THREAD

**Discussion by tazztone** [glibsonoran/Plush-for-ComfyUI#125] [Ideas]
i would LOVE to have a setting for "Style": "manual" that will put a text field where u can write your own style (=system prompt) ( .... or have an LLM generate one, based on onebuttonprompt input ;) 
cheers!

**Comment by glibsonoran** [comment]
I'm not sure I understand.  Maybe you are using the style_prompt node and would like to be able to provide your own system prompt?  If so the Advanced Prompt Enhancer will let you do this, you'll have complete control over the system prompt, prompt and any context or examples you want to add.

**Reply by tazztone** [reply]
thanks
yes you understood right.
looks like i was using the wrong node then sorry
and thanks again for the custom nodes ! :)

**Comment by glibsonoran** [comment]
this is being closed
